pkg/xds/v2/xdsconfig.go

```diff
@@ -299,3 +299,14 @@ func (c *ADSConfig) closeADSStreamClient() {
 	c.StreamClient.Client = nil
 	c.StreamClient = nil
 }
+
+// if got
+// [xds] [ads client] get resp timeout: rpc error: code = ResourceExhausted desc = grpc: received message larger than max (5193322 vs. 4194304), retry after 1s
+// this log, should change this size
+const xdsMaxRecvMsgSize = 4 * 1024 * 1024
+
+func generateDialOption() grpc.DialOption {
+	return grpc.WithDefaultCallOptions(
+		grpc.MaxCallRecvMsgSize(xdsMaxRecvMsgSize),
```

Comment:
It would be better to add an xDS configuration item.

Reply:
Set with gRPC default `math.MaxInt32`